contracts/interfaces/USSSpokePoolInterface.sol

```diff
@@ -38,6 +38,28 @@ interface USSSpokePoolInterface {
         int256 payoutAdjustmentPct;
     }
 
+    struct USSRelayerRefundLeaf {
```

Comment:
Copied exactly from `RelayerRefundLeaf` plus two params at the end. Wish I could use the following syntax:

```
struct USSRelayerRefundLeaf is RelayerRefundLeaf 

or

struct USSRelayerRefundLeaf {
...RelayerRefundLeaf,
}
```